### ISSUE TITLE
Destination indicator and line

### ISSUE THREAD

**Issue by Orrielel** [OpenPoGo/OpenPoGoWeb#71]
Title pretty much says it all.
Either a circle that's bigger than the pokestop icon or a dot that's not intrusive for the eye, and maybe a slightly transparent line towards the circle/dot using `google.maps.Polyline()`. [[Google Dev](https://developers.google.com/maps/documentation/javascript/examples/polyline-simple)]

Idea just popped, but I'll see if I can spare some time to do it tomorrow.

EDIT: I guess the Polylines can be used to incorporate a repeating route feature to the bot

**Comment by Saturate** [comment]
Yeah, I don't like the pokemon trainer icons, they seems a bit to much.

**Comment by Reaver01** [comment]
I don't even know what you are asking.

**Comment by Orrielel** [comment]
An icon indicating what position the bot is walking to, specifically what Pokestop.
Don't know how much use it would be for -m poke.

**Comment by BobbyWibowo** [comment]
It may be possible if we can get data of where the bot is heading to. In the bot (I'm using the dev branch of PokemonGoF-Bot), it outputs which fort the bot is heading to.

Either way, this may require WebSocket implementation. Unless the bot decides to store where it's heading to in the JSON file.
